iidm/iidm-api/src/main/java/com/powsybl/iidm/network/VoltageLevel.java

```diff
@@ -367,6 +368,13 @@
          */
         int[] getNodes();
 
+        /**
+         * Get the list of nodes in bus.
+         */
+        default int[] getNodes(String busId) {
```

Comment:
It could be strange to mention busId in the NodeBreakerView as there is no bus there. Do you add this method in the correct view?